timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -34,38 +35,19 @@
 logger = logging.getLogger("autogluon.timeseries.trainer")
 
 
-# TODO: This class is meant to be moved to `core`, where it will likely
-# TODO: be renamed `AbstractTrainer` and the current `AbstractTrainer`
-# TODO: will inherit from this class.
-# TODO: add documentation for abstract methods
-class SimpleAbstractTrainer:
-    trainer_file_name = "trainer.pkl"
-    trainer_info_name = "info.pkl"
-    trainer_info_json_name = "info.json"
-
+# TODO: This class will be removed after the behavior is reconciled with core
```

Comment:
So this class currently only contains methods that will be completely removed in a follow-up PR?